orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -129,18 +129,18 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 			EWASInferrer.setAltRefDbToFalse();
 		}
 		createAndSetSpeciesInstance(speciesName);
-		setSummationInstance();
+		setSummationInstance(threshold);
 		setEvidenceTypeInstance();
 		OrthologousEntityGenerator.setComplexSummationInstance();
 
 /**
  *  Start of ReactionlikeEvent inference. Retrieves all human ReactionlikeEvents, and attempts to infer each for the species.
  */
 		// Gets DB instance of source species (human)
-		Collection<GKInstance> sourceSpeciesInst = (Collection<GKInstance>) dbAdaptor.fetchInstanceByAttribute("Species", "name", "=", "Homo sapiens");
+		Collection<GKInstance> sourceSpeciesInst = (Collection<GKInstance>) dbAdaptor.fetchInstanceByAttribute("Species", "name", "=", refSpeciesName);
 		if (sourceSpeciesInst.isEmpty())
 		{
-			logger.info("Could not find Species instance for Homo sapiens");
+			logger.info("Could not find Species instance for reference species");
```

Comment:
Should the _name_ of the reference species be used here?

Reply:
Ideally. Just need to pass it in from a global config. I added it to the shell script, but we could pull it from the Species instance, too. Preference?

Reply:
grabbing from config.properties now